src/luv.c

```diff
@@ -436,6 +440,10 @@ static const luaL_Reg luv_udp_methods[] = {
   {"try_send", luv_udp_try_send},
   {"recv_start", luv_udp_recv_start},
   {"recv_stop", luv_udp_recv_stop},
+#if LUV_UV_VERSION_GEQ(1, 27, 0)
+  {"connect", luv_udp_connect},
```

Comment:
Shouldn't this be udp_connect and udp_getpeername on the next line?

Reply:
Those are the metatable bindings for uv_udp_t. So it's for `handle:connect()`, `handle:getpeername()`. The prefixed versions are here: https://github.com/luvit/luv/pull/318/files#diff-8792371db05ee9c0a8a1edb4d4f421acR183